src/app/Fake.FluentMigrator/Fake.FluentMigrator.fsproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
Removed netstandard1.6 as FluentMigrator does not support it (at least the newest version)

Reply:
Fine by me.